File: 图片分享/springboot（后端）/share/src/main/java/com/img/share/dao/Img.java
package com.img.share.dao;

import java.sql.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Img {
    private Integer iid;
    private String iname;
    private String isrc;
    private Date uploaddate;
    private User user;
    private Integer pageview;
}

File: 图片分享/nestjs（管理系统后端)/img-share-manage/src/entity/img.ts
import { BaseEntity, Column, Entity, JoinColumn, ManyToMany, ManyToOne, OneToMany, PrimaryGeneratedColumn } from "typeorm";
import { User } from "./user";

@Entity('img')
export class Img extends BaseEntity {
    @PrimaryGeneratedColumn()
    iid: number
    @Column("varchar", { length: 100 })
    iname: string
    @Column("varchar", { length: 100 })
    isrc: string
    @Column("datetime")
    uploaddate: string
    @ManyToOne(() => User)
    @JoinColumn({
        name: 'uid',
        referencedColumnName: 'uid'
    })
    uid: any
    @Column({ default: 0 })
    pageview: number

    @ManyToMany(() => User, (users: any) => users.imgs)
    users: any[]

    @OneToMany('Comment', 'iid')
    comments: any[]
}